lib/Backends/CPU/libjit/libjit_conv.cpp

```diff
@@ -249,16 +260,21 @@ void libjit_convDKKC8_f(float *outW, const float *inW, const float *filterW,
     // Later we will accumulate values into this slice.
     libjit_conv_init_output_with_bias(n, outW, biasW, outWdims, biasWdims);
 
-    // For each output channel, process [numDepthRegs x float8] elements.
-    for (size_t d = 0; d < outWdims[3]; d += 8 * numDepthRegs * depthStrips) {
+    // For each group of input channels:
+    for (size_t g = 0; g < group; g++) {
+
+      // For each output channel, process [numDepthRegs x float8] elements.
+      for (size_t d = g * outCperG; d < (g + 1) * outCperG;
```

Comment:
Please extract the complex expressions and give them a name:

// Calculate the  .... 
size_t startChannelIndex =  g * outCperG;
size_t endChannelIndex =  (g + 1) * outCperG;